integration-tests/src/test/java/oracle/weblogic/kubernetes/ItAddNewDynamicClusterUsingWlst.java

```diff
@@ -0,0 +1,183 @@
+  private static final int replicaCount = 2;
+  private static final String wlSecretName = "weblogic-credentials";
+  private static String domainNamespace = null;
+
+  private static LoggingFacade logger = null;
+
+  /**
+   * Install operator.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+
+    // get a unique operator namespace
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    String opNamespace = namespaces.get(0);
+
+    // get unique namespaces for Weblogic domain
+    logger.info("Getting unique namespace for domain-on-pv domain");
+    assertNotNull(namespaces.get(1));
+    domainNamespace = namespaces.get(1);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+  }
+
+  /**
+   * Create a domain in PV domain, use an online WLST script to create a new dynamic cluster in the domain,
+   * Patch the domain with a different introspectVersion to trigger introspector
+   * and then verify that server from new cluster is started successfully.
+   */
+  @Test
+  @DisplayName("Create WebLogic domain in PV domain, add a dynamic cluster and "
+      + "verify that server from new cluster is started successfully")
+  void testDynamicClusterNotAsConfigCluster() {
+    // create a domain in PV domain
+    Domain domain = createDomainOnPvUsingWdt(domainUid, domainNamespace, wlSecretName,
+        clusterName, replicaCount, ItAddNewDynamicClusterUsingWlst.class.getSimpleName());
+    assertDomainNotNull(domain);
+
+    // get admin service node port
+    logger.info("Getting node port for default channel");
+    int serviceNodePort = assertDoesNotThrow(() -> getServiceNodePort(
+        domainNamespace, getExternalServicePodName(adminServerPodName), "default"),
+        "Getting admin server node port failed");
+
+    // In OKD cluster, we need to get the routeHost for the external admin service
+    String routeHost = createRouteForOKD(getExternalServicePodName(adminServerPodName), domainNamespace);
+    
+    logger.info("Validating WebLogic admin server access by login to console");
+    testUntil(
+        assertDoesNotThrow(() -> {
+          return adminNodePortAccessible(serviceNodePort, ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT, routeHost);
+        }, "Access to admin server node port failed"),
+        logger,
+        "Console login validation");
+
+    // create a new dynamic cluster using an online WLST script
+    createNewDynCluster();
+
+    // verify the managed server pod in newly added dynamic cluster comes up
+    checkPodReadyAndServiceExists(newManagedServerPodPrefix + 1, domainUid, domainNamespace);
+  }
+
+  private void createNewDynCluster() {
```

Comment:
Modify method name to createNewDynamicCluster()

Reply:
changed